spec/integration/analytics_data_spec.rb

```diff
@@ -41,7 +40,7 @@ def test_fetches_rows_of_analytics_dimensions
     assert_equal [expected_row], rows.to_a
   end
 
-  def test_missing_data_is_nil
+  it "missing_data_is_nil" do
```

Comment:
It seems a shame to leave the test names with broken english....

Maybe we could dump all the names into spreadsheet, challenge people to fill in better names, then do a big search and replace?